### PR TITLE
fix: Fix typos on documentation and comments

### PR DESCRIPTION
**Pull request by criadoperez**
## Description

Text correction on the following files:

```
README.md
src/utils/formatterUtils/formatterUtilsDefinitions.ts
src/components/button/button.tsx
```

### CODE REVIEW

**Review by RuggeroCino — COMMENTED** [review]
Thanks for the fixes @criadoperez , please update the `CHANGELOG` file with a new entry under a `fixed` header like:
```
### Fixed

-   Typos on documentation of comments
```

**Comment by criadoperez** [comment]
Added the changelog as requested.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [1.0.16] - 2024-02-24
+
+### Fixed
+
+-   Typo on `README.md` file
+-   Typo on `src/components/button/button.tsx` file
+-   Typo on `src/utils/formatterUtils/formatterUtilsDefinition.ts` file
```
Comment (cgero-eth) [inline CHANGELOG.md:16]: There's a conflict with the `CHANGELOG` file, also just one entry for the fix would be enough:
```
-   Typos on documentation and comments
```

Comment (criadoperez) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [1.0.17] - 2024-02-27
+
+### Fixed
```
Comment (cgero-eth) [inline CHANGELOG.md:12]: The `1.0.17` header will be added automatically during the next release, just keep the changes under the `unreleased` header. More info in [keepachangelog](https://keepachangelog.com/en/1.1.0/#effort) website
```suggestion
### Fixed
```

**Review by cgero-eth — APPROVED** [review]
Thanks for the fixes

**Comment by cgero-eth** [comment]
@criadoperez We can't merge the PR yet as all commits need to be signed, more info [here](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)

**Pull request closed without merging**